File: src/main/java/fr/spoutnik87/musicbot_rest/constant/PermissionEnum.java
package fr.spoutnik87.musicbot_rest.constant;

import lombok.Getter;

@Getter
public enum PermissionEnum {
    CREATE_MEDIA("CREATE_MEDIA"),
    DELETE_MEDIA("DELETE_MEDIA"),
    READ_MEDIA("READ_MEDIA"),
    CHANGE_MODE("CHANGE_MODE"),
    PLAY_MEDIA("PLAY_MEDIA"),
    STOP_MEDIA("STOP_MEDIA"),
    CREATE_CATEGORY("CREATE_CATEGORY"),
    DELETE_CATEGORY("DELETE_CATEGORY");

    private String value;

    PermissionEnum(String value) {
        this.value = value;
    }
}

File: src/main/kotlin/fr/spoutnik87/musicbot_rest/constant/PermissionEnum.kt
package fr.spoutnik87.musicbot_rest.constant

enum class PermissionEnum(
        val value: String
) {
    CREATE_CONTENT("CREATE_CONTENT"),
    DELETE_CONTENT("DELETE_CONTENT"),
    READ_CONTENT("READ_CONTENT"),
    CHANGE_MODE("CHANGE_MODE"),
    PLAY_MEDIA("PLAY_MEDIA"),
    STOP_MEDIA("STOP_MEDIA"),
    CREATE_CATEGORY("CREATE_CATEGORY"),
    DELETE_CATEGORY("DELETE_CATEGORY")
}